### PR TITLE
Add possible API changes as suggested tags in the sidebar Navigator

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 91762298

## Summary

This PR adds available API change types as tags to the Navigator. Only types, that yield results will be available to the users to pick.

## Dependencies

NA

## Testing

Not yet testable, rely on Unit Tests

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: src/components/Navigator/NavigatorCard.vue [hunk 1]
```diff
@@ -337,8 +352,14 @@ export default {
         // check if `title` matches the pattern, if provided
         const titleMatch = filterPattern ? filterPattern.test(title) : true;
         // check if `type` matches any of the selected tags
-        const tagMatch = selectedTags.length
-          ? tagsSet.has(TOPIC_TYPE_TO_TAG[type]) : true;
+        let tagMatch = true;
+        if (selectedTags.length) {
+          tagMatch = tagsSet.has(TOPIC_TYPE_TO_TAG[type]);
+          // if there are API changes and there is no tag match, try to match change types
+          if (apiChanges && !tagMatch) {
```
Comment (hqhhuang) [inline src/components/Navigator/NavigatorCard.vue:359]: Why do we only try to match change types when there are API Changes and no tag match? 
I was thinking maybe we should always match both Topic and API Changes? So we can further filter the content down.

**Review by hqhhuang — APPROVED** [review]
API Changes are only available for symbol type pages, right? So once a `changeType` tag is selected, we don't need to worry about `Topictype` tags?

**Comment by dobromir-hristov** [comment]
We allow only only one tag to be picked at once. So if you pick `Added` you cannot pick `Article` for example. For now...

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Pull request merged**